src/Microsoft.ML.InternalStreams/InternalStreams.cs

```diff
@@ -6086,9 +6086,9 @@ public long Length
     /// </list>
     /// <para>
     /// Compression support relies on executable utilities to be in the path.
-    /// See <see href="http://7-zip.org"/> for 7z.exe and 7za.exe (for many formats -
-    /// .7z, .gz, .zip, .rar, .bz2, .cab, .arj), <see href="http://gnuwin32.sourceforge.net/packages/gzip.htm"/> for gzip.exe
-    /// (for .gz), or <see href="http://rarsoft.com"/> for unrar.exe (for .rar).
+    /// See <a href="http://7-zip.org">http://7-zip.org</a> for 7z.exe and 7za.exe (for many formats -
```

Comment:
:memo: More cases where `<see href` was removed.

Reply:
see the resp on the other comment. Not officially documented as supported. 

---
In reply to: [202774984](https://github.com/dotnet/machinelearning/pull/529#discussion_r202774984) [](ancestors = 202774984)